### ISSUE TITLE
Work paper

### ISSUE THREAD

**Issue by Jdavidmn** [a23879529/VDSSnet#2]
Hello, I cannot find your work in any paper related to your VDSSnet, I would love to read your work if it has not been published yet, do you have any related reference or article?

**Comment by a23879529** [comment]
Hi Jdavidmn,
this paper hasn't been published yet, but I would like to hand you a copy
of it.
Wish you have a nice day :)

Jdavidmn ***@***.***> 於 2023年2月25日 週六 上午12:11寫道：

**Comment by Jdavidmn** [comment]
That would be awesome! How can you send me the paper? Email perhaps?

**Comment by a23879529** [comment]
Yes, by email, plz give me your email address so I can send you a copy of
the paper.

Jdavidmn ***@***.***> 於 2023年2月25日 週六 09:54 寫道：

**Comment by Jdavidmn** [comment]
Great! Here it is:

j.davidmn9@gmail.com

**Comment by shiningspark** [comment]
hello! I want the paper too. Could you send the copy of the paper to my email: 1821406378@qq.com

**Comment by shiningspark** [comment]
thanks！！

**Comment by genie666** [comment]
hello! I want the paper too. Could you send the copy of the paper to my email: 971360143@qq.com

**Comment by genie666** [comment]
hello! I want the paper too. Could you send the copy of the paper to my email: [971360143@qq.com](mailto:971360143@qq.com)
thank you!

**Comment by shiningspark** [comment]
哪个呀 我记不得这是什么时候的了



---Original---
From: ***@***.***&gt;
Date: Mon, Nov 13, 2023 15:46 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [a23879529/VDSSnet] Work paper (Issue #2)




 
hello! I want the paper too. Could you send the copy of the paper to my email: ***@***.***
